CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.0-beta 
+
+__Rename library to angular_components.__ 
```

Comment:
Put a link here, too. This package now exports from link_to_angular_comp...use that instead...